examples/meanio/README.md

```diff
@@ -0,0 +1,265 @@
+
+
+make sure that the mongodb pod is running and listening by running:
+
+```shell
+
+$ kubectl log meanio-mongodb-controller-nyxxv
+
+```
+
+### Step Three: Create the meanio pod
+
+This is a nodejs server that is configured to talk to the MognoDB server. 
+Note that you will need the MongoDB server up and running before running the meanio pod.
+
+The pod is described in the file `examples/meanio/meanio-controller.json`:
+
+```js
+{
+  "id": "meanio-controller",
+  "kind": "ReplicationController",
+  "apiVersion": "v1beta1",
+  "desiredState": {
+    "replicas": 1,
+    "replicaSelector": {"name": "meanio"},
+    "podTemplate": {
+      "desiredState": {
+         "manifest": {
+           "version": "v1beta1",
+           "id": "meanio",
+           "containers": [{
+             "name": "meanio",
+             "image":"shaiweinstein/k8s:v1",
+             "cpu": 100,
+             "memory": 2000000000,
+             "ports": [{"name": "meanio-server", "containerPort": 3000}],
+             "env": [{"name": "DB_PORT_27017_TCP_ADDR", "value": "meanio-mongodb" }]
+           }]
+         }
+       },
+       "labels": {
+         "name": "meanio",
+         "uses": "meanio-mongodb",
+         "app": "meanio"
+       }
+      }},
+  "labels": {"name": "meanio"}
+}
+```
+
+Using this file, you can turn up your meanio framework  with:
+
+```shell
+$ kubectl create -f examples/meanio/meanio-controller.json
+meanio-controller
+
+$ kubectl get rc
+CONTROLLER                             CONTAINER(S)            IMAGE(S)                                   SELECTOR                     REPLICAS
+meanio-controller                      meanio                  shaiweinstein/k8s:v1                name=meanio                  1
+meanio-mongodb-controller              meanio-mongodb          mongo:2.6.8                         name=meanio-mongodb          1
+
+```
+
+Once that's up (it may take a few minutes to create the pod) you can list the pods, to verify that the mongodb and meanio are running:
+
+```shell
+$ kubectl get pods
+POD                                          IP                  CONTAINER(S)            IMAGE(S)                                   HOST                                                              LABELS                                                     STATUS
+meanio-controller-zwk1b                      10.244.0.13         meanio                  shaiweinstein/k8s:v1                kubernetes-minion-yxea.c.mean-io.internal/104.155.36.70   app=meanio,name=meanio,uses=meanio-mongodb                                  Running             23 hours
+meanio-mongodb-controller-nyxxv              10.244.0.11         meanio-mongodb          mongo:2.6.8                         kubernetes-minion-yxea.c.mean-io.internal/104.155.36.70   app=mongodb,name=meanio-mongodb                                             Running             23 hours
+
+```
+
+You will see a mongodb server pod, and meanio pod.
+
+### Step Four: Create the meanio service.
```

Comment:
Nit: We're not totally consistent on this, but I think we're trying to not end headers with periods